### DISCUSSION TITLE
What would be production grade database alternative to elastic?

### DISCUSSION THREAD

**Discussion by ju-ef** [jasonish/evebox#289] [General]
In previous (Go) version there was postgres support - [tree/c978d02951cbee60d653ced8d49302c7399d057e/postgres](https://github.com/jasonish/evebox/tree/c978d02951cbee60d653ced8d49302c7399d057e/postgres).
Will it be added eventually to the current one?

I just poking around, and thinking about alternative to elastic:
- [postgres](https://github.com/postgres/postgres)
- [ferretdb](https://github.com/FerretDB/FerretDB)
- [mongodb](https://github.com/mongodb/mongo)
- [meilisearch](https://github.com/meilisearch/meilisearch)
- something better?

For now I'm using sqlite3.

**Comment by jasonish** [comment]
The Postgres support was never actually complete enough to be that useful.

In the list you mention, Postgres would still be the best.

- FerretDB: This is just a Mongo proxy layer to Postgres, so its still Postgres.  Interesting idea though.
- MongoDB: I just don't think it has the query power of SQL, but from what I hear its suited well for production uses these days. If that license works for you.
- Meilisearch: Last I looked at this it really wasn't meant to be for logs, but indexing data you have somewhere else.  Its still not recommended for your primary data (mind you neither is Elastic) so I don't consider it a contender.
- Lots use Elastic as part of the production ELK stacks though, so its good for production as well.

Theres a new contender in this space, https://github.com/openobserve/openobserve/, which I'll take a closer look before re-booting the Postgres support.
